<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1481892346949" stop="1481892347055" version="1.4.6">
    <name>Default Suite</name>
    <title>Default Suite : selTask5[]</title>
    <test-cases>
        <test-case start="1481892346999" stop="1481892347052" status="broken">
            <name>officeTabCase</name>
            <description type="text">test Description</description>
            <failure>
                <message>NoClassDefFoundError: org/openqa/selenium/remote/SessionNotFoundException</message>
                <stack-trace>java.lang.NoClassDefFoundError: org/openqa/selenium/remote/SessionNotFoundException
	at com.codeborne.selenide.WebDriverRunner.&lt;clinit&gt;(WebDriverRunner.java:12)
	at com.codeborne.selenide.impl.Navigator.navigateToAbsoluteUrl(Navigator.java:38)
	at com.codeborne.selenide.impl.Navigator.open(Navigator.java:23)
	at com.codeborne.selenide.Selenide.open(Selenide.java:51)
	at com.codeborne.selenide.Selenide.open(Selenide.java:101)
	at com.selenid.task11.AppTest.officeTabCase(AppTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: java.lang.ClassNotFoundException: org.openqa.selenium.remote.SessionNotFoundException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 34 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testId" value="ID-002"/>
                <label name="story" value="Verify the ability to login to app with valid credentials"/>
                <label name="feature" value="Login"/>
                <label name="host" value="SHUMSKAYAA7"/>
                <label name="thread" value="4116@SHUMSKAYAA7.main"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
